packages/ubuntu_desktop_installer/lib/utils/product_info_extractor.dart

```diff
@@ -0,0 +1,64 @@
+import 'dart:io';
+
+import 'package:file/file.dart';
+import 'package:file/local.dart';
+import 'package:flutter/foundation.dart';
+
+class ProductInfoExtractor {
+  static String _cachedProductInfo = '';
+
+  final FileSystem fileSystem;
+
+  ProductInfoExtractor({
+    this.fileSystem = const LocalFileSystem(),
+  });
+
+  String getProductInfo({
+    String isoPath = '/cdrom/.disk/info',
+    String localPath = '/etc/os-release',
```

Comment:
As far as I can tell, `isoPath` and `localPath` are never overridden, not even in tests, so there is no need for these hardcoded file paths to be parameters, they can be made private constants, or local variables in the scope of the method.